start_bundle_coupled.R

```diff
@@ -419,6 +419,19 @@ for(scen in common){
     cfg_mag$mute_ghgprices_until <- scenarios_coupled[scen, "no_ghgprices_land_until"]
   }
 
+  # Write choice of land-use change variable to config. Use smoothed variable
+  # if not specified otherwise in coupled config, i.e. if the column is missing
+  # completely or if the row entry is empty.
+  if (!("var_luc" %in% names(scenarios_coupled))) {
+    cfg_rem$var_luc <- "smooth"
+  } else if (is.na(scenarios_coupled[scen, "var_luc"])) {
+    cfg_rem$var_luc <- "smooth"
```

Comment:
Is that working? I am always not sure if the evaluation of second condition in the OR statement skipped, if the first one already evaluates to TRUE? Because otherwise (if both conditions are checked, or the second before the first), this will raise an error.

Reply:
Ah, cool. Thanks for the suggestion and the explanation.